ethereum/gateway-eth-ts/src/GatewayTsBase.ts

```diff
@@ -100,7 +107,7 @@ export class GatewayTsBase {
     let gatewayToken: GatewayToken;
 
     if (gatewayTokenAddress) {
-      gatewayToken = this.gatewayTokens[gatewayTokenAddress].tokenInstance;
+      gatewayToken = this.gatewayTokens[gatewayTokenAddress]?.tokenInstance;
```

Comment:
why was the questionmark added here? Not sure if the function is robust to returning `undefined`.